nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -400,6 +400,17 @@ systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/var/www" ];
       The rkt module has been removed, it was archived by upstream.
     </para>
    </listitem>
+   <listitem>
+     <para>
+       The FQDN is now added (in addition to the hostname) to
+       <literal>/etc/hosts</literal>. This fixes the FQDN detection (e.g.
+       <literal>hostname --fqdn</literal>) which is required by some
+       applications. These new entries take precedence over entries from the
```

Comment:
rl-2009 is pretty generic. Probably it might make sense to explain a bit more what you're talking about:

> In addition to the hostname, the full qualified domain name (FQDN), which consists of "${cfg.hostName}" and "${cfg.domain}" is now added to /etc/hosts, to fix the `hostname --fqdn` command and other applications that try to determine the FQDN. These new entries…

I was too lazy to add the docbook tags ;-)